cppser/app.cpp

```diff
@@ -123,6 +135,8 @@ int main() {
     Logfile logfile(logname, std::ios::out | std::ios::app);
     Watchdog watchdog(config);
     Singer singer(config);
+    if (!config["auto_watchdog"])
+        watchdog.pause();

```

Comment:
Pausing before the thread even starts seems strange, but I think this is not a big problem as we use std::atomic_bool instead of a condition variable to communicate.

Reply:
Maybe this is just an issue with the naming of the function. Anyway, this doesn't matter much.